content/en/dashboards/ddsql_editor/reference/statements.md

```diff
@@ -14,9 +14,12 @@ DDSQL is in private beta.
 
 {{< code-block lang="text" >}}
 SELECT [ ALL | DISTINCT ] select_expr, ...
-  [ FROM rel_source EVENT_SEARCH 'some message' USE EVENT_INDEX 'index_name'
-[ JOIN_TYPE rel_source ... [ ON condition | USING (column, ... ) ] ] ... ]
-  [ WHERE condition ]
+[ FROM rel_source
```

Comment:
Several changes here:

1. `EVENT_SEARCH` and `USE EVENT_INDEX` are both separably optional, so additional `[ ]` are added. e.g. `FROM t WHERE …`, `FROM t EVENT_SEARCH 'x'`, and `FROM t USE EVENT_INDEX 'y'` are all valid.
2. `join_type` below is described as `INNER`, `OUTER`, `LEFT`, or `RIGHT`, but the keyword `JOIN` was missing; that keyword is required.
3. I indented the `JOIN` clause under the `FROM` one, to indicate that it can only be used with (and immediately following) a `FROM` clause.